documentation/modules/non-admin-permissions-for-ui.adoc

```diff
@@ -0,0 +1,56 @@
+:_content-type: PROCEDURE
+
+[id="non-admin-permissions_{context}"]
+= Granting permissions to non-administrators to work with migration plan components
+
+As an administrator, you can work with all components of migration plans (for example, providers, network mappings, and migration plans).
+
+By default, non-administrators have limited ability to work with migration plans and their components. You can modify their roles to allow them full access to all components or you can give them limited permissions.
+
+For example, you can grant non-administrators the following set of permissions:
+
+* Create and modify storage maps, network maps, and migration plans for the namespaces they have access to.
+* Attach providers created by adminstrators to storage maps, network maps, and migration plans.
+
+[cols="3", options="header"]
+.Permissions required for non-adminstrators to work with migration plan components but not create providers
+|===
+|Actions |API groups |Resources
+
+|All
+|`forklift.konveyer.io`
+|`plans`
+
+|All
+|`forklift.konveyer.io`
+|`migrations`
+
+|All
+|`forklift.konveyer.io`
+|`hooks`
+
+|`get`, `list`, `watch`
+|`forklift.konveyer.io`
+|`providers`
+
+|All
+|`forklift.konveyer.io`
+|`networkmaps`
+
+|All
+|`forklift.konveyer.io`
+|`storagemaps`
+
+|`get`, `list`, `watch`
+|`forklift.konveyer.io`
+|`forkliftcontrollers`
+|===
+
+.Procedure
+
+* To grant cluster-wide permissions, use `ClusterRoleBinding` to add the permissions previously described to a `ClusterRrole` and bind them to the user.
```

Comment:
I'd drop this part in favor of a description of the predefined cluster roles, see my comment above